exercises/concept-wip/wellingtons-weather-station/.docs/instructions.md

```diff
@@ -0,0 +1,95 @@
+# Instructions
+
+The weather station of Wellington is going to replace its manual temperature recording system with a new digital one.
+In the old system, the calculations were performed manually.
+In the new system, they will be automated.
+The calculations include converting the temperature from Celsius to Fahrenheit, and finding the average temperature.
+The new system has sensors to record temperature data in degrees Celsius.
+
+The Wellington weather station wants you to help them with the new system.
+
+## 1. Convert the temperature from Celsius to Kelvin
+
+The weather station has a section on its website that is specified for researchers.
+The researchers are interested in the temperature in Kelvin.
+
+Kelvin is a unit of measurement for temperature.
+It has the same scale as Celsius, but with a different starting point.
+Its starting point is absolute zero, which is 0K (Kelvin) or -273.15°C (Celsius).
+
+Here is a table showing the conversion between Celsius and Kelvin:
+
+| Celsius(°C) | Kelvin(K) |
+| ----------- | --------- |
+| -273.15     | 0         |
+| -100        | 173.15    |
+| 0           | 273.15    |
+| 100         | 373.15    |
+
+The weather station wants you to implement a method that converts a temperature from Celsius to Kelvin.
+
+Implement the `to_kelvin` method that takes an argument `celsius` and returns the temperature in Kelvin by adding 273.15 to the given `celsius` value.
+
+```crystal
+Temperature.to_kelvin(0)
```

Comment:
If you want to keep Temperature as a class, then you need to fix the examples in the intructions
```suggestion
Temperature.new.to_kelvin(0)
```

Reply:
Yeah, this is a valuable point, I will have to look over it a bit more since all of the currently written concepts would need to be needed to fit this style in that case. Out of the classes concept which is being worked on since that one actually uses oop.

Reply:
Okay, it has been applied to all existing concepts, thanks for taking note.